docs/src/conceptual/solution-selection-catalogs.rst

```diff
@@ -4,79 +4,85 @@
 
 .. _solution-selection-catalogs:
 
-***************************
-Solution selection catalogs
-***************************
-
-Tensile provides a mechanism by which only a subset of the code object files produced during a build are loaded at runtime. 
-This is necessary to avoid the overhead associated with loading code object files including initialization time and the 
-memory footprint of the loaded code object files. However, this introduces the problem of knowing which code object file to load. 
-Solution selection is the process by which the **TensileHost** library determines what kernel is preferred and, in turn, 
-what code object file contains the selected kernel. This process uses a hierarchical structure
-to efficiently search for kernels based on hardware, problem size, and transpose, among others. 
-This is the role of the **solution selection catalog** [1]_---a serialized file that uses a hierarchical
-schema to organize kernel metadata for efficient lookup at runtime.
+**************************************
+Tensile's solution selection catalogs
+**************************************
+
+To avoid the overhead associated with loading code object files including initialization time and the memory footprint of the loaded code object files,
+Tensile provides a mechanism to load only a subset of the code object files produced during a build, at runtime.
+To achieve this, it must be determined which code object file to load.
+To determine the preferred kernel and the code object file containing the selected kernel,
+the ``TensileHost`` library utilizes a process named `Solution selection`.
+This process uses a hierarchical structure to efficiently search for kernels based on hardware, problem size, and transpose, among others.
+
+For efficient lookup at runtime, the kernel metadata must be organized in a hierarchical schema in a serialized file named `solution selection catalog` [1]_.
 
 .. note::
-    Throughout this document we will refer to catalog files with the .yaml extension. In practice,
+    This document refers to the catalog files with .yaml extension. In practice,
     solution selection catalogs are usually serialized with `MessagePack <https://msgpack.org/>`_, which uses the .dat extension.
 
-Catalog hierarchy
-=================
+Solution selection catalog hierarchy
+=====================================
 
 .. figure:: ../../assets/msl.svg
     :alt: Master Solution Library hierarchy
     :align: center
 
     Solution selection catalog heirarchy for gfx900 and gfx90a
```

Comment:
```suggestion
    Solution selection catalog hierarchy for gfx900 and gfx90a
```

Reply:
The solution selection catalog is general irrespective of architecture. As an example, gfx900 and 90a are used in the image. Hence, I don't think its a great idea to add the architectures to the heading.